runtime/runtime-params-estimator/Cargo.toml

```diff
@@ -60,11 +60,12 @@ no_cache = [
 ]
 wasmtime = ["near-vm-runner/force_wasmtime"]
 nightly_protocol = ["near-primitives/nightly_protocol"]
-nightly_protocol_features = ["protocol_feature_alt_bn128"]
+nightly_protocol_features = ["protocol_feature_alt_bn128", "protocol_feature_chunk_nodes_cache"]
```

Comment:
Could you remove all `protocol_feature_chunk_nodes_cache` occurrences? I guess it got here from the old chunk cache PR, the feature is stable now.

Reply:
Should be fixed now